packages/ag-charts-enterprise/src/features/status-bar/statusBar.ts

```diff
@@ -252,7 +252,7 @@ export class StatusBar
         this.labelGroup.visible = false;
 
         this.destroyFns.push(
-            ctx.scene.attachNode(this.labelGroup, 'titles'),
+            ctx.scene.attachNode(this.labelGroup),
```

Comment:
This was hacked in to fix https://ag-grid.atlassian.net/browse/CRT-394 - can you verify that this case still works after this change?

Reply:
Verified and works as expected